ports/tests/sample_data/category B/port-B1/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcode 1.0
+version             1.0.0
+
+name                port-B1
+categories          categoryB
```

Comment:
You seem to be using spaces in folder name. That almost sounds like a bad idea, and is probably also inconsistent with category (and `port lint` would usually complain about it as well).